File: Course_01/Java Language/Vetores/negativos.java
package Vetores;

import java.util.Locale;
import java.util.Scanner;

public class negativos {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int n, i;
		
		System.out.print("Quantos numeros você vai digitar? ");
		n = sc.nextInt();
		
		int [] v = new int [n];
		
		for (i = 0; i < n; i++) {
			System.out.print("Digite um número: ");
			v[i] = sc.nextInt();
		}
		System.out.println("NÚMEROS NEGATIVOS: ");
		for (i = 0; i < n; i++) {
			if (v[i] < 0) {
				System.out.println(v[i]);
			}
		}
		
		
		sc.close();
	}

}

File: Course_01/Python/Vetores/Negativos.py
n = int(input("Quantos números você vai digitar? "))

vet = [0 for x in range(n)]

for i in range(0, n):
    vet[i] = float(input("Digite um número: "))
print("Números negativos:")
for i in range(0, n):
    if vet[i] < 0:
        print(vet[i])